package.json

```diff
@@ -45,5 +45,8 @@
     "mustache": "^4.2.0",
     "playwright": "^1.34.2",
     "puppeteer": "^20.1.0"
+  },
+  "dependencies": {
+    "undici": "^5.22.1"
```

Comment:
Do we really need this? `undici` is an additional [1.12 MB](https://www.npmjs.com/package/undici), and we're trying to keep the CLI, where this package is imported, as light as possible.

I think that for the three files that we need to download we could just take the existing code that uses `https`, wrap it in a function and use that one, no need to add another package.

And once we drop Node 16, we can start using the built-in `fetch` available since Node 18.

Reply:
Good point, given we should drop node 16 later this year anyway.

> and we're trying to keep the CLI, where this package is imported, as light as possible.

I'd say the CLI really fat already, so this note sounds a bit funny :]

```
➜  htdocs npx howfat apify-cli
Dependencies: 434
Size: 40.1mb
Files: 12424
```

The dependency graph is really crazy:

https://npm.anvaka.com/#/view/2d/apify-cli

Reply:
Just the oclif core takes almost 10mb, and it has dependency on ts-node -> the whole TS.

```
➜  htdocs npx howfat @oclif/core
Dependencies: 111
Size: 9.69mb
Files: 1609
```

Reply:
I'm trying to get them to drop it, but they're ignoring me 😄 https://github.com/oclif/core/pull/643